mne/viz/epochs.py

```diff
@@ -927,7 +927,11 @@ def plot_epochs(
         if not hasattr(event_id, "keys"):
             event_id = dict()
         # TODO: when min py=3.9, change to `epochs.event_id | event_id`
-        event_id = dict(**epochs.event_id, **event_id)
+        # Let passed-in event_id take precedence (and avoid repeated kwargs)
```

Comment:
Does `event_id = epochs.event_id.copy().update(event_id)` not work? The comprehension isn't pretty but I guess if it's the only way that works in 3.8 there's little we can do.

Reply:
Yeah that should work, I'll push that

Reply:
Turned out not to be as trivial as I thought because of some corner cases found thanks to TDD